Warning in CMap with recent versions of our Debian_Testing test platform

```
[...]/test/Combinatorial_map/Combinatorial_map_test.cpp:168:19: warning: variable 'mark' set but not used [-Wunused-but-set-variable]
   Map1::size_type mark=0;
                   ^~~~
```
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.10-I-192/Combinatorial_map/TestReport_lrineau_Debian-Testing.gz

--- a/Combinatorial_map/test/Combinatorial_map/Combinatorial_map_test.cpp
+++ b/Combinatorial_map/test/Combinatorial_map/Combinatorial_map_test.cpp
@@ -179,6 +179,7 @@ bool test_get_new_mark()
   if ( !res )
   {
       std::cerr<<"PB we can reserve NB_MARK+1 !! mark, exit."<<std::endl;
+      map.free_mark(mark); // This is never supposed to occur.
       return false;
   }
   

